internal/cmd/controller/controllers/clusterregistration/controller.go

```diff
@@ -16,7 +16,6 @@ import (
 	"github.com/rancher/fleet/internal/config"
 	fname "github.com/rancher/fleet/internal/name"
 	"github.com/rancher/fleet/internal/registration"
-	fleetgroup "github.com/rancher/fleet/pkg/apis/fleet.cattle.io"
```

Comment:
This package (see `pkg/apis/fleet.cattle.io/zz_generated_register.go` below) only contained one constant. Hence I directly used the group name defined in the versioned package, as it's also available there.